specification/workloads/resource-manager/Microsoft.Workloads/SAPVirtualInstance/preview/2023-10-01-preview/SAPVirtualInstance.json

```diff
@@ -0,0 +1,4455 @@
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "SAPApplicationServerInstances_Create": {
+            "$ref": "./examples/sapapplicationinstances/SAPApplicationServerInstances_Create.json"
+          },
+          "Create SAP Application Server Instances for HA System with Availability Set": {
+            "$ref": "./examples/sapapplicationinstances/SAPApplicationServerInstances_Create_HA_AvSet.json"
+          }
+        },
+        "description": "Puts the SAP Application Server Instance resource. <br><br>This will be used by service only. PUT by end user will return a Bad Request error.",
+        "parameters": [
+          {
+            "$ref": "../../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SAPVirtualInstanceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SAPApplicationInstanceNameParameter"
+          },
+          {
+            "$ref": "../../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The SAP Application Server Instance resource request body.",
+            "schema": {
+              "$ref": "#/definitions/SAPApplicationServerInstance"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Describes the created SAP Application Server Instance resource.",
+            "schema": {
+              "$ref": "#/definitions/SAPApplicationServerInstance"
+            }
+          },
+          "200": {
+            "description": "Describes the updated SAP Application Server Instance resource.",
+            "schema": {
+              "$ref": "#/definitions/SAPApplicationServerInstance"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "SAPApplicationServerInstances"
+        ],
+        "operationId": "SAPApplicationServerInstances_Update",
+        "x-ms-examples": {
+          "SAPApplicationServerInstances_Update": {
+            "$ref": "./examples/sapapplicationinstances/SAPApplicationServerInstances_Update.json"
+          }
+        },
+        "description": "Puts the SAP Application Server Instance resource.",
```

Comment:
Rephrase the description as
Updates the SAP Application server instance resource. This can be used to update tags on the resource.